cmake/MacOSX/fix_dependencies.rb

```diff
@@ -107,13 +107,13 @@ def extractInstallName(otool_output)
 ###############################################################################
 def fixLoadPath(oldpath,libpath,target)
   debug "#{$install_name_tool} -change #{oldpath} #{$executableId}#{libpath} #{target}"
-  `#{$install_name_tool} -change #{oldpath} #{$executableId}#{libpath} #{target}`
+  `#{$install_name_tool} -change #{oldpath} #{$executableId}#{libpath} #{target} 2>/dev/null`
 end
 
 ###############################################################################
 def fixId(target, libname)
   debug "#{$install_name_tool} -id #{$executableId}#{libname} #{target}"
-  `#{$install_name_tool} -id #{$executableId}#{libname} #{target}`
+  `#{$install_name_tool} -id #{$executableId}#{libname} #{target} 2>/dev/null`
```

Comment:
Good idea but you might lose errors if the file is missing etc.

Reply:
Yeah, this is a temporary fix I'll try to deploy something with `sed` to ignore the warnings that we know about ahead of time, and pass the rest on to stdout.